src/axes/axis.ts

```diff
@@ -19,7 +19,7 @@ export class Axis<D> extends Component {
   protected _baseline: d3.Selection<void>;
   protected _scale: Scale<D, number>;
   private _formatter: Formatter;
-  private _orientation: string;
+  protected _orientation: string;
```

Comment:
No need to make this `protected` -- see `Axes.Time` for an example of how to restrict orientations without accessing `_orientation`.